### PR TITLE
chore: bump dcos-ui-update-service package

### PR DESCRIPTION
**Pull request by brandonc**
## High-level description

Bumps dcos-ui-update service in order to removes ZK creds from logs by allowing alternative configuration of same via ENV and config file

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-46083](https://jira.mesosphere.com/browse/DCOS-46083) Configure the ui-update service with a config file instead of CLI Flags

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): https://github.com/dcos/dcos-ui-update-service/compare/dd453d90603ccf25f73eda45ff0f33550399501a...brandonc/viper
  - [x] Test Results: https://jenkins.mesosphere.com/service/jenkins/job/frontend/job/dcos-ui-update-service/job/PR-28/4/display/redirect
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

File: packages/dcos-ui-update-service/buildinfo.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "single_source": {
     "kind": "url",
-    "url": "http://downloads.mesosphere.com/dcos-ui-update-service/latest/dcos-ui-update-service-afde1262ebf60aeb3b8431ad00d478d63633ae70",
-    "sha1": "f20e4736b6086c57baafb7febb617809fe3f09a9"
+    "url": "http://downloads.mesosphere.io/dcos-ui-update-service/latest/dcos-ui-update-service-8b688f245fe2ce035bc66162155591824da7b96c",
```
Comment (TattdCodeMonkey) [inline packages/dcos-ui-update-service/buildinfo.json]: this should be `.com` instead of `.io` too.

Comment (brandonc) [reply packages/dcos-ui-update-service/buildinfo.json]: ✅

**Comment by brandonc** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by nLight** [comment]
@mesosphere-mergebot sync

**Comment by nLight** [comment]
@mesosphere-mergebot sync

**Comment by nLight** [comment]
@mesosphere-mergebot help

**Comment by nLight** [comment]
@mesosphere-mergebot help

**Comment by nLight** [comment]
@mesosphere-mergebot sync

**Comment by nLight** [comment]
@mesosphere-mergebot label [Ship it]

**Comment by nLight** [comment]
@mesosphere-mergebot override-status mergebot/enterprise/has_ship-it https://jira.mesosphere.com/browse/DCOS-50827

**Comment by orsenthil** [comment]
@mesosphere-mergebot sync

**Pull request merged**